### PR TITLE
Support for zone-redundant storage (ZRS) managed disks

### PR DESCRIPTION
**Pull request by yyvess**
Related to https://github.com/vmware-tanzu/velero/issues/4608

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,5 +18,7 @@ debug
 .vs
 
 .idea/
+/*.iml
 
-_tiltbuild
\ No newline at end of file
+_tiltbuild
+/coverage.out
```
Comment (jenting) [inline .gitignore]: This file is not related to this PR description change.

**Review by jenting — CHANGES_REQUESTED** [review]
go module files are out of date, please commit the changes to go.mod and go.sum

File: .gitignore (hunk 1 above)
Comment (yyvess) [reply .gitignore]: Change removed from this PR

**Review by jenting — APPROVED** [review]

**Comment by reasonerjt** [comment]
@ywk253100 please review.

**Review by ywk253100 — APPROVED** [review]
